internal/cmd/scraper/scraper.go

```diff
@@ -97,6 +95,11 @@ func validateConfig(cfg *Config) error {
 		}
 	}
 
+	if cfg.WorkerThreads < 4 {
```

Comment:
Does it make sense to set as well a maximum? Es 1000?

Reply:
I don't think it's be necessary, but we should add a comment explaining what the setting does.
The min-check was mostly to prevent people from accidentally setting it to zero